examples/exp_configs/rl/multiagent/multiagent_i210.py

```diff
@@ -47,7 +47,7 @@
     "human",
     num_vehicles=0,
     lane_change_params=SumoLaneChangeParams(lane_change_mode="strategic"),
-    acceleration_controller=(IDMController, {"a": .3, "b": 2.0, "noise": 0.6}),
+    acceleration_controller=(IDMController, {"a": .3, "b": 2.0, "noise": 0.5}),
```

Comment:
different PR pls

Reply:
I don't understand, these changes are already merged into the other branch